spec/models/report_spec.rb

```diff
@@ -576,6 +576,41 @@
         expect(row[:topic_id]).to eq(post.topic.id)
       end
     end
+
+    context "with editor filter" do
+      let(:editor_with_one_edit) { Fabricate(:user) }
+      let(:editor_with_two_edits) { Fabricate(:user) }
+
+      let(:report_with_one_edit) do
+        Report.find('post_edits', { filters: { 'editor_id' => editor_with_one_edit.id } })
+      end
+
+      let(:report_with_two_edits) do
+        Report.find('post_edits', { filters: { 'editor_id' => editor_with_two_edits.id } })
+      end
+
+      let(:posts) { Fabricate.times(3, :post) }
+
+      before do
+        freeze_time
+
+        2.times do |i|
+          posts[i].revise(editor_with_two_edits, raw: "edit #{i + 1}")
+        end
+
+        posts.last.revise(editor_with_one_edit, raw: 'edit 3')
+      end
```

Comment:
```suggestion
      fab!(:posts) { Fabricate.times(3, :post) }

      fab!(:editor_with_two_edit) do 
        Fabricate(:user).tap do |user|
          2.times do |i|
            posts[i].revise(user, raw: "edit #{i + 1}")
          end
        end
      end

      fab!(:editor_with_one_edit) do 
        Fabricate(:user).tap do |user|
          posts.last.revise(user, raw: "edit 3")
        end
      end

      let(:report_with_one_edit) do
        Report.find('post_edits', { filters: { 'editor_id' => editor_with_one_edit.id } })
      end

      let(:report_with_two_edits) do
        Report.find('post_edits', { filters: { 'editor_id' => editor_with_two_edits.id } })
      end
```

Small suggestion for the fabrication to make ti clearer.

Reply:
thanks for the suggestion, this is my first time using fabrication. I always use factory bot